src/components/resize-listener-mixin.cjsx

```diff
@@ -19,7 +19,7 @@ module.exports =
     @resizeListener = _.throttle(@resizeEffect, @state.resizeThrottle or @props.resizeThrottle)
 
   componentDidMount: ->
-    @setInitialSize()
+    _.delay(@setInitialSize, 0)
     window.addEventListener('resize', @resizeListener)
```

Comment:
You could just use `_.defer` here.  It's almost exactly the same (but uses 1ms).  http://underscorejs.org/docs/underscore.html#section-81